language-support/qute/qute.tmLanguage.json

```diff
@@ -80,17 +86,45 @@
 			"name": "string.unquoted.cdata.qute"
 		},
     "section_start_tag": {
-      "begin": "({)(#)(else\\sif|\\w+(\\.\\w+)*)",
+      "begin": "({)(#)(\\w+(\\.\\w+)*)",
       "end": "(\\/)?((?<![\\\\])})",
       "beginCaptures": {
         "1": {
           "name": "support.constant.handlebars"
         },
         "2": {
-          "name": "keyword.control"
+          "name": "entity.name.function.hash"
         },
         "3": {
+          "name": "entity.name.function.identifier"
+        }
+      },
+      "endCaptures": {
+        "1": {
           "name": "keyword.control"
+        },
+        "2": {
+          "name": "support.constant.handlebars"
+        }
+      },
+      "patterns": [
+        {
+          "include": "#code"
+        }
+      ]
+    },
+    "section_core_start_tag": {
+      "begin": "({)(#)(each|else|else\\sif|eval|for|if|include|insert|set|let|with|switch|case|is|when)",
+      "end": "(\\/)?((?<![\\\\])})",
+      "beginCaptures": {
+        "1": {
+          "name": "support.constant.handlebars"
+        },
+        "2": {
+          "name": "keyword.control"
```

Comment:
The colour of `keyworkd.control` is magenta, while `entity.name.function.hash` is blue. I decided to go with purple for the core tag.